bentoml/yatai/deployment/aws_ec2/constants.py

```diff
@@ -0,0 +1,5 @@
+MIN_CAPACITY = 1
+DESIRED_CAPACITY = 1
+MAX_CAPACITY = 1
+INSTANCE_TYPE = "t2.micro"
```

Comment:
rename:  INSTANCE_TYPE => DEFAULT_INSTANCE_TYPE

Reply:
Changed all to default.